src/global/tracking/IterativeVertexFinder_factory.h

```diff
@@ -26,6 +27,7 @@ class IterativeVertexFinder_factory :
     std::unique_ptr<AlgoT> m_algo;
 
     Input<ActsExamples::Trajectories> m_acts_trajectories_input {this};
+    Input<edm4eic::ReconstructedParticle> m_edm4eic_reconParticles_input {this, "ReconstructedParticles"};
```

Comment:
```suggestion
    PodioInput<edm4eic::ReconstructedParticle> m_edm4eic_reconParticles_input {this};
```
Not sure how a `const char*` converts to `InputOptions`, but it seems like this second argument is not even needed.